plugins/PrivacyManager/angularjs/opt-out-customizer/opt-out-customizer.directive.html

```diff
@@ -5,41 +5,75 @@
     </p>
 
     <h3>Customize the Opt-out iframe</h3>
-        <div>
-            <p>
-                <span>
-                Font Color:
-                <input type="color" ng-model="optOutCustomizer.fontColor" ng-change="optOutCustomizer.onUpdate()">
-                </span>
-
-                <span>
-                Background Color:
-                <input type="color" ng-model="optOutCustomizer.backgroundColor" ng-change="optOutCustomizer.onUpdate()">
-                </span>
-
-                <span>
-                Font Size:
-                <input id=FontSizeInput type="number" min="1" max="100" ng-model="optOutCustomizer.fontSize" ng-change="optOutCustomizer.updateFontSize()">
-                </span>
-
-                <span>
-                        <select class="browser-default" ng-model="optOutCustomizer.fontSizeUnit" ng-change="optOutCustomizer.updateFontSize()">
-                                <option value="px">px</option>
-                                <option value="pt">pt</option>
-                                <option value="em">em</option>
-                                <option value="rem">rem</option>
-                                <option value="%">%</option>
-                        </select>
-                </span>
-
-                <span>
-                Font Family:
-                <input id=FontFamilyInput type="text" ng-model="optOutCustomizer.fontFamily" ng-change="optOutCustomizer.onUpdate()">
-                </span>
-                
-            </p>
+    <div>
+        <p>
+            <span>
+            Font Color:
+            <input type="color" ng-model="optOutCustomizer.fontColor" ng-change="optOutCustomizer.onUpdate()">
+            </span>
+
+            <span>
+            Background Color:
+            <input type="color" ng-model="optOutCustomizer.backgroundColor" ng-change="optOutCustomizer.onUpdate()">
+            </span>
+
+            <span>
+            Font Size:
+            <input id=FontSizeInput type="number" min="1" max="100" ng-model="optOutCustomizer.fontSize" ng-change="optOutCustomizer.updateFontSize()">
+            </span>
+
+            <span>
+                    <select class="browser-default" ng-model="optOutCustomizer.fontSizeUnit" ng-change="optOutCustomizer.updateFontSize()">
+                            <option value="px">px</option>
+                            <option value="pt">pt</option>
+                            <option value="em">em</option>
+                            <option value="rem">rem</option>
+                            <option value="%">%</option>
+                    </select>
+            </span>
+
+            <span>
+            Font Family:
+            <input id=FontFamilyInput type="text" ng-model="optOutCustomizer.fontFamily" ng-change="optOutCustomizer.onUpdate()">
+            </span>
+
+        </p>
+    </div>
+
+    <h3>Customize opt-out text</h3>
+    <div>
```

Comment:
@diosmosis when I used it, it wasn't clear to me what the "customise opt-out text" does in privacy manager.  eg does it append it as URL parameter, is it only for showing it, is it saving it (but there is no save button which we usually always have), etc. I think it be better to remove it from this screen, or only show it disabled as a hint of what the text is. This makes it also easier when there is only one place where it can be edited. 

Also only super users should be able to change that but admin users can view this page so for them it wouldn't work.

Reply:
Removed.